include/entity.h

```diff
@@ -221,25 +221,28 @@ typedef struct {
 } ET_WeaponUnk006;
 
 typedef struct PACKED {
-    /* 0x7C */ s16 lifetime;
+    /* 0x7C */ s16 unk7C;
```

Comment:
Do the previous names still make sense with the new function decompiled? I imagine this is just because the scratch was created before the renames happened

Reply:
I removed the old names because they didn't seem to apply to this weapon — looks like the struct was just copied & pasted before.